bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -0,0 +1,241 @@
+
+{% block body_id %}firefox-os{% endblock %}
+{% block body_class %}firefox-os{% endblock %}
+
+{# Do not load CSS bundle specified in base template (duplicate CSS). #}
+{% block site_css %}{% endblock %}
+
+{% block extrahead %}
+  {{ super() }}
+  {% stylesheet 'firefox_os_new' %}
+<!--[if lt IE 9]>
+  {% stylesheet 'firefox_os_ie' %}
+<![endif]-->
+{% endblock %}
+
+{% block site_header %}{% endblock %}
+
+{% block site_header_unwrapped %}
+  {% include 'firefox/includes/family-nav.html' %}
+{% endblock %}
+
+{% block content %}
+  <main role="main">
+    <section class="fxos-hero-space">
+      <div class="content-wrapper">
+        <div class="phone">
+          <div class="fxos-cta">
+            <h1><span>{{_('Firefox <span>OS<span>')}}</span></h1>
+            <p class="tagline">{{_('Designed to give you just what you need, just when you need it.')}}</p>
+
+            <div id="primary-cta-signup" class="signup hidden">
+              <a href="#email-form-wrapper" class="cta-button get-newsletter newsletter-signup-toggle" role="button" aria-haspopup="true">{{_('Sign me up')}}</a>
+              <p class="signup-txt">{{_('Sign up for our monthly newsletter and get the latest on Firefox OS')}}</p>
+            </div>
+            <a id="primary-cta-phone" href="#get-device" class="cta-button get-phone hidden" role="button" aria-haspopup="true">{{_('Get a phone')}}</a>
+          </div>
+
+          <div href="#" class="phone-container">
+            <img src="{{ static('img/firefox/os-new/device_static.png') }}" width="1030" height="782" alt="{{_('Firefox OS device illustration')}}" />
+
+            <a href="#" id="trigger-homescreen" class="try-homescreen" role="button">
+              <!-- TODO: descriptive content to be added post March 1 for a11y -->
+              <div class="throbber throb-one"></div>
+              <div class="throbber throb-two"></div>
+              <div class="throbber throb-three"></div>
+              <div class="throbber throb-four"></div>
+            </a>
+
+            <a href="#" id="trigger-swipe" class="try-swipe" role="button">
+              <!-- TODO: descriptive content to be added post March 1 for a11y -->
+              <div class="throbber throb-one"></div>
+              <div class="throbber throb-two"></div>
+              <div class="throbber throb-three"></div>
+              <div class="throbber throb-four"></div>
+            </a>
+          </div>
+        </div>
+
+        <div class="demo">
+          <h2>{{_('Give these a try')}}</h2>
+          <a href="#" id="close-demo" class="close">
+            <span>{{_('Close')}}</span>
+          </a>
+          <div class="stage scroll-homescreen" data-button="#scroll-homescreen">
+            <img src="{{ static('img/firefox/os-new/device.png') }}" width="304" height="637" alt="cellphone outlines" />
```

Comment:
This has hard-coded English alt text, but if the images are largely presentational an empty alt attribute might suffice?